sql/branch.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+
+    DROP VIEW current_payment_instructions;
```

Comment:
The view includes the `participant` and `team` columns (`SELECT [...] *`), so if we don't drop those columns here then the next steps will fail.

```
[gratipay] $ psql gratipay-test
Null display is "¤".
Line style is unicode.
Border style is 2.
psql (9.4.5, server 9.3.2)
Type "help" for help.

gratipay-test=# alter table payment_instructions drop column participant;
ERROR:  cannot drop table payment_instructions column participant because other objects depend on it
DETAIL:  view current_payment_instructions depends on table payment_instructions column participant
HINT:  Use DROP ... CASCADE to drop the dependent objects too.
gratipay-test=#
```